### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by matthewtiscareno**
## Summary

Release dictionary with v1.0.0

## Test Data and/or Report

Enter info here

## Related Issues

Enter related issues here

### CODE REVIEW

**Comment by mit3ch** [comment]
This looks ready to build. There is one minor set of edits that you should consider fixing before the public release. Before each of the classes, there is a descriptive comment (e.g., ) in which "attributes" should be replaced with "class".

**Review by mace-space — APPROVED** [review]
Subject to Mitch's (minor) edits, this is approved

**Comment by matthewtiscareno** [comment]
> There is one minor set of edits that you should consider fixing before the public release. Before each of the classes, there is a descriptive comment (e.g., "-- Observing program attributes --") in which "attributes" should be replaced with "class".

Done, I think.

File: src/PDS4_HST_IngestLDD.xml
Comment (markshowalter) [inline src/PDS4_HST_IngestLDD.xml:1]: Lines 194-195 fix wording:
`For HST instruments that do not have multiple channels, such as WFPC2, this
attribute has the same value as the instrument_id attribute.`

Starting at line 555, there's no clear explanation of how `observation_type` gets a value of "TIME-SERIES", and there were also a few other issues. I recommend this revision:
```
            The type of the observation, one of IMAGING, SPECTROSCOPIC, or TIME-SERIES.
            IMAGING indicates that a 2-D image with two spatial axes was obtained.
            SPECTROSCOPIC indicates that one of the axes of the data array is related to
            wavelength. TIME-SERIES indicates that one of the axes is sampling time.

            For many HST data files, this is the value of FITS keyword OBSTYPE. Where
            OBSTYPE does not appear, this value depends on other information.
            - All ACS, NICMOS, WFC3, WFPC, and WFPC2 observations are IMAGING.
            - All COS, FOS, and GHRS observations are SPECTROSCOPIC.
            - All HSP observations are TIME-SERIES.
            - For FOC, the observation_type depends on the value of FITS keyword OPMODE:
              IMAGING if OPMODE is "ACQ" or "IMAGE"; TIME-SERIES of OPMODE is "OCC";
              SPECTROSCOPIC if OPMODE is "SPEC".

            Note that, for ACS/HRC, the FITS keyword OBSTYPE can have a value of
            CORONAGRAPHIC. However, the value of this attribute is IMAGING; the
            coronagraph_flag attribute indicates whether the coronagraph was used.
```

**Review by markshowalter — CHANGES_REQUESTED** [review]
My comments are just revisions of two definitions. Somebody should look them over but then they should be suitable for cut-and-paste into the document. I mentioned on Slack that `moving_target_flag` should probably not be nillable. Everything else looks good.

**Comment by matthewtiscareno** [comment]
Per @markshowalter's review, I have done the following: 
- I added his suggested text for `channel_id` and `observation_type`.  For the latter, I changed his first comma into a colon, and I also changed the last sentence so that now reads as follows: 
`In such cases, the value of this attribute is IMAGING and the coronagraph_flag attribute indicates whether the coronagraph was used.`
- I added a corresponding sentence to `coronagraph_flag` explaining the same concept.
- I removed nillability from `moving_target_flag`
- I added sentences to the definitions of `hst_quality_id` and `hst_quality_comment`, as we're doing for other nillable attributes, explaining when they might be nil and which value of `nilReason` should be used.

You should be able to see all of this in the diff.  Please review before we proceed with the merge.

**Pull request merged**